drake/util/drakeGeometryUtil.h

```diff
@@ -531,6 +531,17 @@ Eigen::Matrix<typename Derived::Scalar, 3, 3> rpy2rotmat(
       s(2) * s(1) * c(0) - c(2) * s(0);
   R.row(2) << -s(1), c(1) * s(0), c(1) * c(0);
 
+  // Clean up the rotation matrix by treating any value less than the system's epsilon as zero.
```

Comment:
The `for` loop below was commented out temporarily while @jwnimmer-tri and I tried to chase a Travis CI Server error. I previously found that they are needed to ensure comparison test between prius.urdf and prius.sdf is successful. 

Since the `for` loop below is an obvious hack, I will remove it but implement a different way of comparing matrices using relative tolerances rather than absolute tolerances (see: [discussion about relative tolerance](https://github.com/RobotLocomotion/drake/issues/1887#issuecomment-198800478)).